test/remote/token_code_tests.js

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const assert = require('insist')
+const config = require('../../config').getProperties()
+const TestServer = require('../test_server')
+const Client = require('../client')()
+const error = require('../../lib/error')
+
+describe('remote tokenCodes', function () {
+  let server, client, email, code
+  const password = 'pssssst'
+  const metricsContext = {
+    flowId: '0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef',
+    flowBeginTime: Date.now()
+  }
```

Comment:
ISTM the tests should assert something about this `metricsContext`, e.g. that the correct flow-id header was present in the outgoing email.